src/main/java/gameframework/motion/GameMovable.java

```diff
@@ -3,10 +3,18 @@
 import java.awt.Point;
 
 public abstract class GameMovable implements Movable {
-	protected GameMovableDriver moveDriver = new GameMovableDriverDefaultImpl();
+	protected GameMovableDriver moveDriver ;
 	protected Point position = new Point();
 	protected SpeedVector speedVector = SpeedVector.createNullVector();
 
+	public GameMovable(){
+		moveDriver = new GameMovableDriverDefaultImpl();
```

Comment:
please call the other constructor instead of copying the implementation. This will make it easier for subclasses to override the behavior of this class.